src/Attributes/SnakeToCamelCase.php

```diff
@@ -5,6 +5,15 @@
 use Attribute;
 
 #[Attribute]
-class SnakeToCamelCase
+class SnakeToCamelCase implements CaseConversionInterface
 {
+    public function convertToCase(string $input): string
```

Comment:
Dont change the parameter name during inheritance. If it's `$string` on top level, keep it `$string` everywhere else too.

Reply:
I think that was 2 AM coding mistake...